CONTRIBUTING.md

```diff
@@ -72,28 +41,57 @@ for ways to improve the API to make it easy to work with the GitHub API.
 
 ## Making Changes
 
-When you're ready to make a change, 
-[create a branch](https://help.github.com/articles/fork-a-repo#create-branches) 
-off the `master` branch. We use `master` as the default branch for the 
+When you're ready to make a change,
+[create a branch](https://help.github.com/articles/fork-a-repo#create-branches)
```

Comment:
I rewrote this whole section to just have CLI instructions (and ensure they have latest `master`). Let me know how it feels to you.